.github/workflows/deploy_batcher_contract.yml

```
@@ -0,0 +1,180 @@
+name: Deploy Batcher Contract and Update Release
+on:
+  release:
+    types: [published]
+jobs:
+  lint-and-test:
+    environment: dev
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

```suggestion
      - uses: actions/checkout@v4
```
Please use the latest version of the action.